pages/runtime/supervisor-api.md

```diff
@@ -0,0 +1,349 @@
+# Interacting with the Resin Supervisor
+
+The Resin Supervisor is resin.io's agent that runs on devices. Its main role is to ensure your app is running, and keep communications with the Resin API server.
```

Comment:
may be worth adding a warning that only images downloaded after some date will have this API exposed, at least until supervisor updates are happening often.